src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/AbstractPlatformProxy.cs

```diff
@@ -190,5 +199,11 @@ public virtual IMsalHttpClientFactory CreateDefaultHttpClientFactory()
         {
             return new SimpleHttpClientFactory();
         }
+
+        /// <summary>
+        /// On Android, iOS and UWP, MSAL will save the legacy ADAL cache in a known location.
```

Comment:
Can this be done with #if "<Platform>" and define the value(true/false) based on the platform instead of making it virtual and have each platform override it?

Reply:
Yea, but generally the pattern in MSAL is for platform specific behavior go into each proxy.